modules/console/pages/flow-console.adoc

```diff
@@ -0,0 +1,114 @@
+include::{mod-loc}attributes.adoc[]
+ifndef::mod-loc[]
+:mod-loc: ./partials/
+endif::mod-loc[]
+//Category: flow-console
+// Type: assembly
+[id="skupper-console"] 
+= Using the Skupper console
+
+include::{mod-loc}preface.adoc[]
+
+The Skupper console provides data and visualizations of the traffic flow between Skupper sites.
+
+// Type: procedure
+[id="enabling-console"] 
+== Enabling the Skupper console
+
+By default, when you create a {skupper-name} site, you do not the Skupper console.
+
+When enabled, the Skupper console URL is displayed whenever you check site status using `skupper status`.
+
+.Prerequisites
+
+* A Kubernetes namespace where you plan to create a site
+
+.Procedure
+
+. Determine which site in your {service-network} is best to enable the console.
++
+--
+Enabling the console also requires that you enable the flow-collector component, which requires resources to process traffic data from all sites.
+You might locate the console using the following criteria:
+
+* Does the {service-network} cross a firewall?
+For example, if you want the console to be available only inside the firewall, you need to locate the flow-collector and console on a site inside the firewall.
+
+* Is there a site that processes more traffic than other sites?
+For example, if you have a _frontend_ component that calls a set of services from other sites, it might make sense to locate the flow collector and console on that site to minimize data traffic.
+
+* Is there a site with more or cheaper resources that you want to use?
+For example, if you have two sites, A and B, and resources are more expensive on site A, you might want to locate the flow collector and console on site B.
+--
+
+. Create a site with the flow collector and console enabled:
++
+----
+$ skupper init --enable-console --enable-flow-collector
+----
++
+NOTE: You must enable the console on the same site as you enable the flow collector.
```

Comment:
I would like to put this info in Bold and as a first info under the title " Create a site with the flow collector and console enabled". In my opinion this Is an important info more than a note.

Reply:
Promoted to 'important', would be against style guide to bold it all.. 
I don't think it makes sense to move it from here because:
* not explaining what the flow collector is
* not explaining the design decision around this
The scope of this doc is 'console', the flow collector is an implementation detail from a user POV